source/Octostache/Templates/TemplateParser.cs

```diff
@@ -3,7 +3,11 @@
 using System.Collections.Specialized;
 using System.Globalization;
 using System.Linq;
+#if NET46
 using System.Runtime.Caching;
+#else
+using Microsoft.Extensions.Caching.Memory;
```

Comment:
I considered that... Ended up on the fence. Worth discussing more.
I'll investigate the cache size limiting.